Gems/ScriptAutomation/Code/Source/Platform/Windows/Utils_Windows.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <Utils/Utils.h>
+
+#include <AzCore/PlatformIncl.h>
+
+namespace ScriptAutomation
+{
+    namespace Utils
+    {
+        bool SupportsResizeClientArea()
+        {
+            return true;
+        }
```

Comment:
Instead of adding a new PAL function in ScriptAutomation, consider adding it to WindowBus, right next to ResizeClientArea(), similar to CanToggleFullScreenState(). That is a better place for it, and would save you from adding 10 new files.

See 
https://github.com/o3de/o3de/blob/1e948814d00b10fdc2950bb8cf330ee62eb3b16f/Code/Framework/AzFramework/AzFramework/Windowing/WindowBus.h#L68

Reply:
I updated to move the query function to the WindowBus.